CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 * Remove 'Request can not be moved above page request' dialog. Part of UIREQ-379.
 * Provide reorder queue permission. Refs UIREQ-318.
 * Show reorder view after moving request from one item to another. Refs UIREQ-334.
+* Show effective call number in details pain. Refs UIREQ-366.
```

Comment:
I think it's a typo - "pain". :)

Reply:
Ha! A [Del McCoury moment](https://youtu.be/080Kwc1WWiQ?t=102). Thanks!